docs/pfsense_installerconfig.md

```diff
@@ -59,19 +135,17 @@ installation is performed.
 exit
 ```
 
-This file must be located in **/etc/** directory.
-
-**/etc/installerconfig** script will install pfSense and replace all
-configuration files that we want to change before first boot.
-
-Examples
---------
+This file must be located in the `etc` directory. The `etc/installerconfig`
+script will install pfSense and replace all configuration files that we want to
+change before the first boot.
 
-Example file, **installerconfig**, is located in **scripts/** directory.
+An example file is located
+[here](https://github.com/pcengines/apu2-documentation/blob/master/scripts/installerconfig).
+It should work without any corrections.
```

Comment:
```suggestion
```
I wouldn't be so sure. The installerconfig may change slightly with the pfSense/FreeBSD versions, I think.